GrowingTextViewHandler/GrowingTextViewHandler/Source/GrowingTextViewHandler.swift

```diff
@@ -8,60 +8,60 @@
 
 import UIKit
 
-public class GrowingTextViewHandler: NSObject {
-	public var growingTextView : UITextView!
-	public var heightChangeAnimationDuration = 0.5
-	public var maximumNumberOfLines = Int.max {
+open class GrowingTextViewHandler: NSObject {
+	open var growingTextView : UITextView!
+	open var heightChangeAnimationDuration = 0.5
+	open var maximumNumberOfLines = Int.max {
 		didSet {
 			updateInitialHeightAndResize()
 		}
 	}
-	public var minimumNumberOfLines = 1 {
+	open var minimumNumberOfLines = 1 {
 		didSet {
 			updateInitialHeightAndResize()
 		}
 	}
 
-	private var textViewHeightConstraint: NSLayoutConstraint?
+	fileprivate var textViewHeightConstraint: NSLayoutConstraint?
```

Comment:
@Anders123fr What is the rationale behind using `filePrivate` instead of `private`? I believe, these variables should be accessible from this declaration only.